### ISSUE TITLE
multiselect column header filter with ajaxdata

### ISSUE THREAD

**Issue by jjech** [olifolkerd/tabulator#2884]
**Describe the bug**
We are using the new multiselect column header filters (tabulator 4.7.1) in an ajaxdata configuration.

When the user selects an item to filter on from the header filter dropdown menu, an ajaxdata call is triggered and the table updates with the new filtered data based on the user's selection. However, the dropdown menu does not highlight / retain the user's selection in the dropdown menu. A subsequent filter action on the column will result in a completely new filter that does not retain the user's previous selection(s).

**Tabulator Info**
4.7.1

**Working Example**
Unfortunately I can't provide a JSFiddle or Codepen for this problem as it requires an ajaxdata configuration. Instead, I have done my best to describe the problem in an accurate way that is hopefully understandable. I am also offering an image illustrating the behavior as described: https://imgur.com/5H3DHpc

**To Reproduce**
(Requires having a tabulator instance running in an ajaxdata configuration with multiselect header filters implemented for at least one column header.)
1. Click a multiselect column header filter to get the new multiselect dropdown menu.
2. Click on an element in the dropdown to trigger a filter and ajaxdata request.
3. The column header filter is populated with the element that was selected, but it is not highlighted in the dropdown menu.
4. Click on another element from the dropdown menu to trigger another filter update / ajaxdata request.
5. The column header filter is populated with the new element that was selected and the previous selection is lost.

**Expected behavior**
Ideally, the multiselect column header filters would retain the user's dropdown selections between ajaxdata requests. Currently, when the new multiselect feature is used in an ajaxdata configuration, it functions as a single item filter only (per column).

**Comment by olifolkerd** [comment]
Hey @jjech 

Could you please post a link to a js fiddle that replicates this, it is very hard to offer advice without seeing how your table or filter is configured.  you can easily mock up a ajax request by pointing to a .json file in the same folder (it is how the ajax examples on the tabulator website do it)

Cheers

Oli :)

**Comment by jjech** [comment]
Sorry @olifolkerd; I didn't realize how easily I could create a JSFiddle using ajaxURL. In hindsight, it was a trivial task --- I just pointed to [a public ajaxURL](https://jsonplaceholder.typicode.com/posts).

The issue I described was caused by not using the correct headerFilterFunc ("in"). I had opened a similar issue #2854 but did not check back to see your comments there.

Here is a JSFiddle of tabulator configured with ajaxURL showing the behavior working as expected when the correct headerFilterFunc is set: https://jsfiddle.net/jjech/63g0kLwx/

Thank you for educating me!
